src/AutoRest.CSharp/LowLevel/Output/LowLevelRestClient.cs

```diff
@@ -48,13 +48,23 @@ private IEnumerable<RestClientMethod> BuildAllMethods()
                 ServiceRequest serviceRequest = operation.Requests.FirstOrDefault(r => r.Protocol.Http is HttpRequest);
                 if (serviceRequest != null)
                 {
+                    // Prepare our parameter list. If there were any parameters that should be passed in the body of the request,
+                    // we want to generate a single parameter of type `RequestContent` named `requestBody` at the start of the
+                    // parameter list instead of creating a seperate parameter for each of them.
+
                     IEnumerable<RequestParameter> requestParameters = serviceRequest.Parameters.Where (FilterServiceParamaters);
                     RestClientMethod method = _builder.BuildMethod(operation, (HttpRequest)serviceRequest.Protocol.Http!, requestParameters, null, true);
-                    // Inject the body parameter
                     List<Parameter> parameters = method.Parameters.ToList();
-                    Parameter bodyParam = new Parameter ("requestBody", "The request body", typeof(Azure.Core.RequestContent), null, true);
-                    parameters.Insert (0, bodyParam);
-                    RequestBody body = new RequestContentRequestBody (bodyParam);
+                    RequestBody? body = null;
```

Comment:
I guess if we have no body params passing a null body is fine here, right? Previously we conditionally replaced it.

Reply:
Yep